recipe/meta.yaml

```diff
@@ -36,9 +35,6 @@ test:
     - mock
     - pytest-mock
     - coverage
-    - pytest
-    - coverage
-    - stone >=2.*
```

Comment:
The unnecessary dependencies were removed form the test section, and the package was tested. 
The package builds properly after removing the unnecessary dependencies.